### PR TITLE
Code update

### PR DESCRIPTION
**Pull request by MaximeAlgosup**
The following pages have been added:
- Profile for the company side
- Swipe for the company side
The following features have been added:
- App translation for all the pages
- Correct basic issues

### CODE REVIEW

File: adopte_a_candidate/lib/pages/sign_up.dart
Comment (Biohazardyee) [inline adopte_a_candidate/lib/pages/sign_up.dart:1]: String? _validateConfirmPassword(String? value, String password) {
    if (value == null || value.isEmpty) {
      return AppLocalizations.of(context)!.confirmPassword;
    }
    if (value != password) {
      return 'Passwords do not match';
    }
    return null;
  }
  
  In this function, for internationalization, we should remplace return 'Passwords do not match'; with the required translation

**Review by LucasAub — APPROVED** [review]
Looks good to me!

File: adopte_a_candidate/lib/pages/splash_screen.dart
Comment (Biohazardyee) [inline adopte_a_candidate/lib/pages/splash_screen.dart:1]: We have twice the same thing basically, splash_screen.dart and loading_screen.dart, we need to discuss which we keep.

File: adopte_a_candidate/lib/widgets/loadings/loading_page.dart
Comment (Biohazardyee) [inline adopte_a_candidate/lib/widgets/loadings/loading_page.dart:1]: Refer to the splash_screen.dart review

**Review by Biohazardyee — CHANGES_REQUESTED** [review]

**Review by Biohazardyee — APPROVED** [review]
Everything looks good to me now

**Pull request merged**